Src/Web/Web.Shared.Net/FilterRequest.cs

```diff
@@ -1,14 +1,14 @@
 ﻿namespace Microsoft.ApplicationInsights.Web
```

Comment:
Public API classes are in Web (like Processors, TelemetryInitializers and Module), everything else is under Implementation. I suspect this class needs to be moved. Could you please also check other classes. E.g. RequestNotificationFilter